### ISSUE TITLE
[A11y] [Issue 11] Selected state not provided within chat

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#3137]
# 11 Severity 2 - Selected state not provided within chat

> This is filed to https://github.com/microsoft/BotFramework-WebChat/issues/3137.

## Background

Combo box for state selection does not announce “Selected” state. If screen reader user selects “Washington”, it only announces “Washington” instead of “Washington, selected.”

## Investigation notes

This is owned by Adaptive Cards. After "Washington" is selected, in their DOM tree, the `<option>` does not have `aria-selected` attribute.

[Bug]

**Comment by compulim** [comment]
Potentially fixed in #3193, will verify.

**Comment by compulim** [comment]
Verified and is fixed in #3193.
